src/arrays.js

```diff
@@ -1,61 +1,74 @@
 const getNthElement = (index, array) => {
-  // your code here
+  let newIndex = index;
+  if (index >= array.length) {
+    newIndex = index - array.length;
+  }
+  return array[newIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  let csvString = array.join(",");
+  return csvString;
 };
 
 const csvStringToArray = string => {
-  // your code here
+  let anArray = string.split(",");
+  return anArray;
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(item => item.toString());
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(item => item.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(item => item.split("").reverse().join(""));
 };
 
 const onlyEven = numbers => {
-  // your code here
+  return numbers.filter(item => item % 2 === 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  const newArray = [...array]; //Make a copy of the original array
+  const modifiedArray = newArray.splice(index, 1); //removes the nth element
+  return newArray; //returns new modified array without removed element
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  return strings.filter(item => item.match(/^[aeiou]/i));
 };
 
 const removeSpaces = string => {
-  // your code here
+  let array= string.split(" ")
+  array.filter(item => item.trim());
+  string = array.join("");
```

Comment:
I am not sure what was the intention here, to reassign a new value to the string, or to assign the value of array.join() to a variable which you have named string.
In the first case, you shouldnt reassign anything to a value what is being passed in as a parameter, if it is the second, you should choose a different name and not the same as the string parameter, and use the let/const keywords.
You could just return it straightaway, dont have to assign it to a variable :), could join everything up like this:
````
const removeSpaces = string => {
  return string
    .split(' ')
    .filter(item => item.trim())
    .join('');
};
````